Dockerfile

```diff
@@ -14,12 +14,10 @@ RUN npm install --no-package-lock
 # Build the application
 RUN npm run build
 
-ENV CLIENT_PORT=6274
-ENV SERVER_PORT=6277
+ARG CLIENT_PORT=6274
+ARG SERVER_PORT=6277
```

Comment:
Doesn't this change preclude setting the port numbers from outside the container at startup?

Reply:
Actually, it's a little confusing because to override the mapped port on the container we need to set an environment variable, but in reality, you don't **need** to specify the port that the inspector uses _in the container_, only on the host, so `-p 1234:6274` is fine, there's no real need to override the `6274` port since that's internal to the container